CONTRIBUTING.md

```diff
@@ -2,32 +2,97 @@
+
+```
+# Fetch all new changes on the cognoma repo
+git fetch upstream
+
+# Make sure you're on your master branch.
+git checkout master
+
+# Fast forward your master branch with upstream changes.
+git merge --no-ff upstream/master
+```
+
+### Pull request advice
+
+1. Focus a pull request on a single feature. Smaller pull requests are easier to review and will be thus get merged quicker. If you want to change multiple things, create multiple pull requests.
++ Open incomplete pull requests but put WIP in the title for "work in progress". This enables early stage feedback to save time later and helps coordinate efforts. Once you feel the pull request is ready, remove WIP from the title.
++ Please help review open pull requests. Even if you don't fully know the topic, the dialogue will often be productive. As a reviewer, it's helpful to note the type of review you performed: did you test the code, look it over, or are you just supporting the concept?
++ Repository maintainers are encouraged to squash merge pull requests when appropriate. This will compact all of the commits into one, heling keep the commit history clean.
+
+**Happy contributing!**
 
-Currently, [members](https://github.com/orgs/cognoma/people "Cognoma People") of the `cognoma` GitHub organization have commit access on all [cognoma repositories](https://github.com/cognoma). However, our goal is to transition to sustainable project development without the direct need for organization involvement. Therefore, organization members will grant (and revoke) commit access on a per repository level. For example, a Django developer working on [`cognoma/django-cognoma`](https://github.com/cognoma/django-cognoma) will receive commit access to `cognoma/django-cognoma` but not necessarily `cognoma/cognoma`. Commit access will be extended based on a history of positive involvement in the project.
+## Maintainers
 
-The primary purpose of commit access is for merging pull requests. The committer should use their best judgment on how how much review is necessary for a given pull request. For some pull requests, a single individual with commit access may be able to fully review a pull request. Commit access may also be used for small or urgent changes as specified by the repository-specific `CONTRIBUTING.md`. If you think Project Cognoma would benefit from granting you commit access, please contact an [organization member](https://github.com/orgs/cognoma/people "Cognoma People"). However, please don't request commit access until you've accumulated a history of involvement in the project.
+Maintainers are individuals with commit access to Cognoma repositories, who are responsible for reviewing and merging pull requests. In addition, maintainers help keep repositories organized and issues up to date. Maintainers are selected based on a sustained, positive history of contribution to Cognoma and mastery of the GitHub-based workflow.
```

Comment:
subject confusion. Should be something like: 
"Maintainers are individuals with commit access to Cognoma repositories and are responsible for reviewing and merging pull requests."